backend/src/controllers/UserController.js

```diff
@@ -0,0 +1,42 @@
+const connection = require('../database/connection');
+
+module.exports = {
+    async index(request, response) {
+        const user = await connection('user').select('*');
+    
+        return response.json(user);
+    },
+
+    async create(request, response){
+        const { username, password, email, active } = request.body;
+
+        try { 
+            await connection('user').insert({
+                username,
+                password,
+                email,
+                active
+            })
```

Comment:
Suggestion: add ";" at end of command line.

Reply:
done.